### PR TITLE
Fix 3d pixel size

### PR DESCRIPTION
**Pull request by nvaytet**
When using the default pythreejs shader for points, it seems we need to scale the pixels up by 2.5 to get the correct size in the scene.

### CODE REVIEW

File: src/scipp/plotting/figure3d.py [hunk 1]
```diff
@@ -201,8 +201,11 @@ def _create_point_cloud(self, positions):
             })
 
         pixel_ratio = config.plot.get("pixel_ratio", 1.0)
+        # Note that an additional factor of 2.5 (obtained from trial and error) seems to
+        # be required to get the sizes right in the scene.
```
Comment (SimonHeybrock) [inline src/scipp/plotting/figure3d.py:205]: Why was this not needed before? In the docs example, will the points overlap now, if we switch to the orginal number of points?

Comment (nvaytet) [reply src/scipp/plotting/figure3d.py:205]: Yes the points will now overlap in the original example. It was not picked up because there is some logic to compute a semi sensible point size if it is not supplied. It is set equal to 0.05 * mean(box_size).

In the original example, the mean was around 40, so points had a size of about 2 (while really the size should have been 1 because the coords are just arange). And without the additional factor in, 2 looked close enough to 2.5.

When you changed the example, the auto computation of pixel size changed, revealing the bug.

Basically the pixel size should have been set to 1 in the example.

There is also an example in the instrument view docs page that checks for the correct size of pixels, but this is in scippneutron and I did not run that...
